data/demographic/census/index.html

```diff
@@ -59,7 +77,36 @@
 {% include abstract.html
 name="Demographic Data"
 stewards="AGRC and U.S. Census Bureau"
-abstract="This data package contains demographic information collected by the Census Bureau in 2010 and 2000. Demographic population includes population counts, household counts, ethnicity counts, etc.  Below, the U.S. Census Bureau Data is the data in the format released by the Census Bureau, with just a few modifications to make the data more user friendly. The Populated Block Areas 2010 Approximation and Populated Place Points 2010 Approximation are datasets derived by AGRC from 2010 Census data." %}
+abstract="This page contains boundary and demographic data collected by the U.S. Census Bureau for 2020, 2010 and 2000 censuses. Demographic data includes population counts, household counts, ethnicity counts, etc. The data available on this page is filtered, geographically, to the state of Utah. This data follows the format released by the Census Bureau, with just a few modifications for usability." %}
+<p>Read about <a href="{% link data/demographic/census/index.html %}#Commonly-Used Census Bureau Geography">commonly-used Census Bureau geography</a> including blocks, block groups, tracts, and places.</p>
```

Comment:
**thought**: i wasn't sure about spaces in anchor tags and i'll assume you tested this (and can dismiss these comments) but if not these two suggestions use a simpler anchor.
```suggestion
<p>Read about <a href="{% link data/demographic/census/index.html %}#common-definitions">commonly-used Census Bureau geography</a> including blocks, block groups, tracts, and places.</p>
```